core/roboconf-dm/src/main/java/net/roboconf/dm/internal/api/impl/TargetsMngrImpl.java

```diff
@@ -442,7 +442,7 @@ public String findTargetId( AbstractApplication app, String instancePath ) {
 		if( targetDir.isDirectory()){
 			List<File> scriptFiles = Utils.listAllFiles(targetDir);
 			for( File scriptFile : scriptFiles) {
-				if( scriptFile.getName().contains(Constants.SCRIPT_NAME)) {
+				if( scriptFile.getName().contains(Constants.SCOPED_SCRIPT_SUFFIX)) {
```

Comment:
You made everything fgood.
However, I missed this one!  :fearful:  When you search for script resources, you must send all the scripts resources. not just those that contain "-script." in their name. There should be only one such file. In a given target directory, all the files should be scripts, except those that have the "properties" extension.

It makes me think you forgot a test where you send several scripts.
One called "main-script.sh" and the other one being "library.sh".